scripts/script.yaml

```diff
@@ -747,44 +780,131 @@
+                      - show: רוב הזמן
+                        value: mostly_not
+                      - show: בחלק קטן מהזמן
+                        value: mostly_yes
+                      - show: לא
+                        value: never
+                      - show: לא רוצה להשיב
+                        value: no_response
+                        class: "other"
+
+                  - say: "ומה עם כפפות?"
+                  - wait:
+                      variable: habits_wear_gloves
+                      options:
+                      - show: כן, כל הזמן
+                        value: always
+                      - show: רוב הזמן
+                        value: mostly_not
+                      - show: בחלק קטן מהזמן
+                        value: mostly_yes
+                      - show: לא
+                        value: never
+                      - show: לא רוצה להשיב
+                        value: no_response
+                        class: "other"
+
+
+                 # - do:
+                 #     cmd: fetch_public_transportation_question_date
+                 #     params:
+                 #       - record
+                  
+                 # - switch:
+                 #     arg: _public_transportation_question_ask
+                 #     cases:
+                 #       match: true
+                 #       steps:
+                  - say: האם יצא לך להשתמש בתחבורה ציבורית בשבוע האחרון?          
+                  - wait:
+                      variable: public_transportation_last_week
+                      options:
+                        - show: כן
+                          value: true
+                          steps:
+                          - say: במה מאלה השתמשת השבוע?
+                          - wait:
+                              variable: _var
+                              multi: true
+                              options:
+                              - show: "רכבת"
+                                field: public_transportation_train
+                              - show: "אוטובוס"
+                                field: public_transportation_bus 
+                              - show: "מונית"
+                                field: public_transportation_taxi
+                              - show: "אחר"
+                                field: public_transportation_other
+                              - show: |-
+                                  <span class='empty'>בעצם לא</span>
+                                  <span class='not-empty'>זהו</span>
+                                  <span class='none-selected'>אין פרטים נוספים</span>
+                                class: other
+                          - do:
+                              cmd: update_from_selection
+                              params:
+                                - record
+                                - _var
+                            
+                        - show: לא
+                          value: none
```

Comment:
@noamoss @akariv 
shouldn't this value be `false`?
does `none` mean the string `none` will be the value?
please explain..

Reply:
This pr is no longer relevant @OriHoch